js/chrome-tabs.js

```diff
@@ -24,7 +31,7 @@
 
     init(el, options) {
       this.el = el
-      this.options = options
+      this.options = extend({}, defaultOptions, options)
```

Comment:
I know this may seem to contradict my [suggestion here](https://github.com/adamschwartz/chrome-tabs/pull/20#issuecomment-287678205) but at the time it wasn’t yet made concrete what would be included in a hypothetical `defaultOptions`.